nav2_behavior_tree/include/nav2_behavior_tree/bt_action_server_impl.hpp

```diff
@@ -285,6 +296,15 @@ bool BtActionServer<ActionT>::loadBehaviorTree(const std::string & bt_xml_filena
   topic_logger_ = std::make_unique<RosTopicLogger>(client_node_, tree_);
 
   current_bt_xml_filename_ = filename;
+
+  // Enable monitoring with Groot2
+  if (enable_groot_monitoring_) {
+    bt_->addGrootMonitoring(&tree_, groot_publisher_port_);
+    RCLCPP_INFO(
```

Comment:
Perhaps a debug log? I don't feel married to this, but feels like maybe that's a little TMI to log each time you call a new navigation action.